crates/sui-indexer-alt-graphql/src/api/mutation.rs

```diff
@@ -21,6 +21,9 @@ pub enum TransactionInputError {
 
     #[error("Invalid signature format in signature {index}: {err}")]
     InvalidSignatureFormat { index: usize, err: FastCryptoError },
+
+    #[error("Invalid JSON transaction data: {0}")]
```

Comment:
```suggestion
    #[error("Invalid JSON-encoded gRPC Transaction: {0}")]
```

Let's be specific about the type that we expected.

Same goes for the BCS error message, try something like:

> Invalid BCS-encoded TransactionData